server/static/css/style.css

```
@@ -1387,3 +1387,238 @@ body.role-operator .role-admin-only {
+    display:inline-flex; align-items:center; gap:0.4rem;
+    padding:0.3rem 0.65rem;
+    background:rgba(255,255,255,0.08); border:1px solid rgba(255,255,255,0.12);
+    border-radius:999px; font-size:0.7rem; font-weight:600; letter-spacing:0.4px;
+    margin-bottom:1rem; color:#cfd6ff;
+}
+.aside-eyebrow .dot {
+    width:6px; height:6px; border-radius:50%; background:#2ed573;
+    box-shadow:0 0 0 0 rgba(46,213,115,0.7);
+    animation: pulseG 2s infinite;
+}
+@keyframes pulseG {
```

Stylelint (17.10.0)

[error] Expected keyframe name "pulseG" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)